xgcm/test/test_grid_ufunc.py

```diff
@@ -445,8 +572,13 @@ def grad_to_inner(a):
         assert_equal(v, expected_v)
 
         # Test decorator
-        @as_grid_ufunc("(X:center,Y:center)->(X:inner,Y:center),(X:center,Y:inner)")
-        def grad_to_inner(a):
+        @as_grid_ufunc()
+        def grad_to_inner(
+            a: Gridded[xr.DataArray, "X:center,Y:center"]
+        ) -> Tuple[
+            Gridded[xr.DataArray, "X:inner,Y:center"],
+            Gridded[xr.DataArray, "X:center,Y:inner"],
+        ]:
```

Comment:
This is a good example of what the syntax looks like for a somewhat complicated function signature.